kuma/wiki/middleware.py

```diff
@@ -33,8 +33,16 @@ def process_request(self, request):
                 ('$subscribe' in request.path or '$files' in request.path)):
             return None
 
+        # Process the request path
+        path = request.path_info
+        request_slug = path.lstrip('/')
+        maybe_lang = request_slug.split(u'/')[0]
+        if maybe_lang in settings.ENABLED_LOCALES:
+            path = u'/' + u'/'.join(request_slug.split(u'/')[1:])
+        else:
+            path = u'/' + request_slug
+
         # Skip slugs that don't have locales, and won't be in a zone
-        request_slug = request.path_info.lstrip('/')
         if any(request_slug.startswith(slug)
```

Comment:
Nit. It seems more efficient to move lines 46-48 immediately after the definition of `request_slug`, and then start the `maybe_lang` and `path` stuff.